xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/user/impl/xwiki/XWikiAuthServiceImpl.java

```diff
@@ -606,4 +669,26 @@ protected String stripContextPathFromURL(URL url, XWikiContext context)
 
         return strippedURL;
     }
+
+    private static final List<Event> LISTENER_EVENTS = Arrays.<Event>asList(new UserAuthenticatedEvent());
+
+    @Override
+    public List<Event> getEvents()
+    {
+        return LISTENER_EVENTS;
+    }
+
+    @Override
+    public void onEvent(Event event, Object source, Object data)
+    {
+        if (event instanceof UserAuthenticatedEvent) {
+            LOGGER.info("Authenticated user [{}]", source.toString());
```

Comment:
Is this some debug leftover ?

Reply:
At the moment yes, since I know I have to implement tests to make it a acceptable PR.